rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -246,9 +251,17 @@ fn start_recv_thread(
             loop {
                 match session.receive_blocking() {
```

Comment:
If you use this `match` as an expression, you can reduce the nesting here:

```rust
let pkt = match session.receive_blocking() { ... }
```

Reply:
Hm, I don't see it. Cause I'm branching on two different errors too